### ISSUE TITLE
Search input focus outline not correct in Safari

### ISSUE THREAD

**Issue by Yapaslfeu** [necolas/normalize.css#547]
After modifying search input with -webkit-appearance: textfield, the focus outline is remaining with outline-offset=0, but should be -2px like other textfield inputs:
[type='search']:focus {
	outline-offset: -2px;
}

**Comment by Yapaslfeu** [comment]
Problem is very visible on Safari

**Comment by Yapaslfeu** [comment]
When i say "modifying search input", i mean on line 383 in normalize.css
This modification breaks the way the field outline is displayed in Safari

**Comment by jonathantneal** [comment]
Good catch. Confirmed by the User Agent stylesheets.
